catalog/MDCCatalog/AppDelegate.swift

```diff
@@ -63,6 +63,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     MDCTabBarColorThemer.apply(colorScheme, to: MDCTabBar.appearance())
     MDCTextFieldColorThemer.applyColorScheme(toAllTextInputControllerDefault: colorScheme)
 
+    do {
```

Comment:
Drop the do {} wrapper and move next to MDCButtonColorThemer above.

Reply:
Alright. I was trying to keep the extra colorScheme's scope limited.